config/private.go

```diff
@@ -10,6 +10,9 @@ type Private struct {
 	// none was found, it's the path to the default config file that
 	// was written.
 	OriginalPath string `json:"-"`
+	// OriginalAPIKeyPath is the original path to the API key in the configuration file. Only used to modify the external KV store in case of API Key Reset.
+	// This is set automatically in afterConfSetup()
+	OriginalAPIKeyPath string `json:"-"`
```

Comment:
lets be clear that this is only used when the gateway is running in slave mode. We can do here 2 things:
- State in the documentation that this is for mdcb envs when the gw is edge
- Or change the name of the var to be explicit